### PR TITLE
[RW-2748][risk=no] Include gfortran on AoU clusters for R support

### PR DESCRIPTION
**Pull request by calbach**
https://precisionmedicineinitiative.atlassian.net/browse/RW-2748

This is a workaround while Leo investigates and fixes in the base docker image: https://broadworkbench.atlassian.net/browse/IA-1002

### CODE REVIEW

File: api/cluster-resources/setup_notebook_cluster.sh [hunk 1]
```diff
@@ -23,15 +23,20 @@ jupyter nbextension enable snippets_menu/main
 # each time the R kernel starts.
 echo "Sys.setenv(RETICULATE_PYTHON = '$(which python3)')" >> ~/.Rprofile
 
-# Required for R summarytools to function. Must be installed from the cran
-# repository, otherwise will receive install errors. See:
-# https://github.com/DataBiosphere/leonardo/issues/813
+# The following are required for R packages to function:
+# - libmagick++-dev: summarytools
+# - g++-4.6-miltilib: minqa (brings in the gfortran library)
```
Comment (jmthibault79) [inline api/cluster-resources/setup_notebook_cluster.sh]: typo: multilib

Comment (calbach) [reply api/cluster-resources/setup_notebook_cluster.sh]: Done, thanks

**Comment by calbach** [comment]
OK, made some fixes after doing some e2e local testing of this. @jmthibault79 please take another look when you have a chance (not urgent - just want to get this in tomorrow at some point).

File: api/cluster-resources/README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ To manually test updates to this script locally:
   local server config to use your custom script:
 
   ```
-  api$ sed -i "s,setup_notebook_cluster\.sh,setup_notebook_cluster-${USER}.sh," config/config_local.json
+  api$ sed -i "s,setup_notebook_cluster\.sh,setup_notebook_cluster-${USER}.sh," src/main/java/org/pmiops/workbench/notebooks/LeonardoNotebooksClientImpl.java
```
Comment (jmthibault79) [inline api/cluster-resources/README.md]: Good catch.

**Pull request merged**